classes/form/AbstractForm.php

```diff
@@ -204,4 +212,27 @@ public function setValue($field_name, $value)
 
         return $this;
     }
+
+    /**
```

Comment:
This method doesn't respect the single responsability principle. It returns a check or a message. Could you fix it, please ?

Reply:
Thanks @PululuK

Reply:
Shame on me 😪

Reply:
Why ?

Reply:
Don't worry ... It's a private joke.
You are too young to understand this 😅

Reply:
Are you sure ? ;)

Reply:
Certainty is uncertain @Progi1984

Reply:
:smile: